sdk/containerregistry/container-registry/src/containerRegistryClient.ts

```diff
@@ -10,26 +12,38 @@ import {
   isTokenCredential
 } from "@azure/core-http";
 import { CanonicalCode } from "@opentelemetry/api";
+import "@azure/core-paging";
```

Comment:
Ah, I see you're already importing core-paging. You can ignore my comment on async-iterator-pollyfill then :smile: